### PR TITLE
Unify formating of strings across modules

### LINKED ISSUE #5660: Unify formatting of similar strings across modules
In order to create more logical UI and ease understanding of editing process I believe we need to unify formatting of similar strings.

__filmic rgb:__
- structure/texture
- bloom/reconstruct
- grey/colorful details
- shadow/highlights details
- middle tones

__color contrast:__
- green vs magenta
- blue vs yellow

__color balance:__
- lift / offset
- gamma / power
- gain / slope
- mid-tones

__color lookup table:__
-  green/red (BTW, why red, not magenta?)
- blue/yellow

### PR DESCRIPTION
**Pull request by johnny-bit**
Fixes #5660 

- make sure to use single spelling - `midtones` for all things dealing with middle tones
- explain color look up table and update labels
- use UTF-8 arrows for labels in filmic

### CODE REVIEW

File: src/iop/colorchecker.c [hunk 1]
```diff
@@ -1327,31 +1327,31 @@ void gui_init(struct dt_iop_module_t *self)
...
 
   g->scale_L = dt_bauhaus_slider_new_with_range(self, -100.0, 200.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_L, _("lightness offset"));
+  gtk_widget_set_tooltip_text(g->scale_L, _("scale lightness in Lab space\nlower values darken target color while higher brighten it"));
   dt_bauhaus_widget_set_label(g->scale_L, NULL, N_("lightness"));
 
   g->scale_a = dt_bauhaus_slider_new_with_range(self, -256.0, 256.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_a, _("chroma offset green/red"));
-  dt_bauhaus_widget_set_label(g->scale_a, NULL, N_("green/red"));
+  gtk_widget_set_tooltip_text(g->scale_a, _("scale target color in Lab a dimmension\nlower values shift target color towards greens while higher shift towards magentas"));
```
Comment (TurboGit) [inline src/iop/colorchecker.c:1334]: Lab a dimension is not easy to read to me. Maybe Lab-a or Lab 'a' ?

File: src/iop/colorchecker.c [hunk 2]
```diff
@@ -1327,31 +1327,31 @@ void gui_init(struct dt_iop_module_t *self)
...
+  dt_bauhaus_widget_set_label(g->scale_a, NULL, N_("green-magenta offset"));
   dt_bauhaus_slider_set_stop(g->scale_a, 0.0, 0.0, 1.0, 0.2);
   dt_bauhaus_slider_set_stop(g->scale_a, 0.5, 1.0, 1.0, 1.0);
   dt_bauhaus_slider_set_stop(g->scale_a, 1.0, 1.0, 0.0, 0.2);
 
   g->scale_b = dt_bauhaus_slider_new_with_range(self, -256.0, 256.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_b, _("chroma offset blue/yellow"));
-  dt_bauhaus_widget_set_label(g->scale_b, NULL, N_("blue/yellow"));
+  gtk_widget_set_tooltip_text(g->scale_b, _("scale target color in Lab b dimmension\nlower values shift target color towards blues while higher shift towards yellows"));
```
Comment (TurboGit) [inline src/iop/colorchecker.c:1341]: Lab b is read ok, but will need to be made consistent with whatever is done for Lab a. above.

File: src/iop/colorchecker.c [hunk 3]
```diff
@@ -1327,31 +1327,31 @@ void gui_init(struct dt_iop_module_t *self)
   dt_color_picker_new(self, DT_COLOR_PICKER_POINT_AREA, g->combobox_patch);
 
   g->scale_L = dt_bauhaus_slider_new_with_range(self, -100.0, 200.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_L, _("lightness offset"));
+  gtk_widget_set_tooltip_text(g->scale_L, _("scale lightness in Lab space\nlower values darken target color while higher brighten it"));
```
Comment (TurboGit) [inline src/iop/colorchecker.c:1330]: To be consistent with below wording:

`scale target in Lab L dimension\n...`

?

File: src/iop/colorchecker.c (hunk 1 above)
Comment (elstoc) [reply src/iop/colorchecker.c:1334]: Also "dimension" only has one "m"

**Review by TurboGit — APPROVED** [review]
Looks good to me now . The new arrows will have to be double checked on Windows where we add issues for some special characters. Or maybe it is a standard symbol chars?

**Comment by johnny-bit** [comment]
I took recommended utf 8 character that has HTML symbol... But in case either Mac or windows don't have them we can revert single commit...

**Comment by elstoc** [comment]
I know I'm a bit late to the party on this one but now that I'm looking at the documentation, I'm really not a big fan of "midtones". Does anyone mind if we change it to "mid-tones"?

**Comment by johnny-bit** [comment]
I certainly don't mind ;) just make it consistent across the src and docs and we're good.

**Comment by elstoc** [comment]
Done: #9858

**Comment by TurboGit** [comment]
Ok, just saw this discussion here. So I'll merge the PR. Thanks.

**Pull request merged**